substrate/frame/revive/fixtures/contracts/base_fee.rs

```diff
@@ -0,0 +1,36 @@
+// This file is part of Substrate.
+
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Not demanding to remove this in principle SPDX already fixes this up, and rest is redundant information.

Reply:
If we want to change the header it needs to be a separate PR changing headers in all files.